src/models/msg.js

```diff
@@ -6,6 +6,15 @@ var id = 0;
 
 class Msg {
 	constructor(attr) {
+		// Some properties need to be copied in the Msg object instead of referenced
+		if (attr) {
+			["from", "invited", "target"].forEach((prop) => {
```

Comment:
If we're going this route, change invited to target in invite messages so that it's somewhat consistent.

Reply:
Gotcha, will do.

Reply:
Done! Would you mind testing a bit to make sure I didn't break invite stuff? I can't seem to be able to display "xxx invited yyy", only "xxx invited you".